recipes/timezonefinder/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "timezonefinder" %}
+{% set version = "1.5.7" %}
+{% set sha256 = "c9de5429aca0d991459d0103e7feaae9c8d0be4516d88018411b775abe5c4579" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - numpy
+    - numba    # [py>27]
```

Comment:
Why is numba only for `>py27`? Numba is supported for py2 and 3 and I don't see any indication in the source repo's README that the library doesn't support numba for py27.